Studio-Lovelies/GG-JointJustice-Unity
# Implemented credits identical to Lua version

## Summary
Implements the credits identical to the Lua version using ~~a markdown file~~ TMP rich text

## Before/after screenshots and/or animated gif
https://user-images.githubusercontent.com/1689033/153987329-a8b9dee9-7713-478a-9a00-6dbac75ca35e.mp4

## Testing instructions
1. Open the `Credits` scene
2. Play the scene and notice it being identical to the Lua version
3. Make changes to the files at `Assets/Resources/Credits/credits.md`
4. Play the scene again and see the changes reflected in the scene

## Additional information

- `[ ]` Changes UI
- `[x]` Introduces new feature
  - Scrolling credits
- `[ ]` Removes existing feature
- `[x]` Has associated resource: 
  - closes #123
